### PR TITLE
Feature/BRPA-759-get-options-middleware

### PR DESCRIPTION
**Pull request by MIzidio**
#### What is the purpose of this pull request?

Moving the getOptions middleware from tiktok-tbp-config to this repository and adding the countries names to countryList return

#### How to test it?

[Workspace](http://tiktok--izidio.myvtex.com/_v/api/tiktok-tbp/options)

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: node/index.ts [hunk 1]
```diff
@@ -34,4 +35,9 @@ declare global {
 // Export a service that defines route handlers and client options.
 export default new Service({
   clients,
+  routes: {
+    getOptions: method({
+      GET: [getOptions],
+    }),
+  },
```
Comment (ConradoClark) [inline node/index.ts:42]: Hmm 🤔 this function is already being exposed through `GraphQL` in the other PR. I don't think we're going to use this route in the front-end.

File: node/middlewares/getOptions.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import type { GetTikTokDataResponse } from '../typings/getTikTokDataResponse'
```
Comment (ConradoClark) [inline node/middlewares/getOptions.ts:1]: Same here. This is already being done by the `externalDataService`.

**Review by ConradoClark — CHANGES_REQUESTED** [review]

File: node/middlewares/getOptions.ts (hunk 2 above)
Comment (MIzidio) [reply node/middlewares/getOptions.ts:1]: So can I delete this file ?

**Comment by felipefadul** [comment]
Unfortunately, I think all of this PR was done in [this PR](https://github.com/vtex-apps/tiktok-tbp/pull/10).
Issues [BRPA-755](https://vtex-dev.atlassian.net/browse/BRPA-755) and [BRPA-759](https://vtex-dev.atlassian.net/browse/BRPA-759) were very similar. Aren't they both done in the [PR-10](https://github.com/vtex-apps/tiktok-tbp/pull/10) and [PR-14](https://github.com/vtex-apps/tiktok-tbp/pull/14)?

**Comment by ConradoClark** [comment]
> Unfortunately, I think all of this PR was done in [this PR](https://github.com/vtex-apps/tiktok-tbp/pull/10).
> Issues [BRPA-755](https://vtex-dev.atlassian.net/browse/BRPA-755) and [BRPA-759](https://vtex-dev.atlassian.net/browse/BRPA-759) were very similar. Aren't they both done in the [PR-10](https://github.com/vtex-apps/tiktok-tbp/pull/10) and [PR-14](https://github.com/vtex-apps/tiktok-tbp/pull/14)?


Yes, I believe so. I ended up doing BRPA-759 with the migration, @MIzidio can change it to done.

File: node/middlewares/getOptions.ts (hunk 2 above)
Comment (ConradoClark) [reply node/middlewares/getOptions.ts:1]: As Fadul mentioned below, I think it's safe to close this PR altogether.

**Pull request closed without merging**